src/software/sensor_fusion/vision_detection.h

```diff
@@ -1,66 +1,46 @@
 #pragma once
 
-#include <vector>
-
 #include "software/new_geom/angle.h"
 #include "software/new_geom/point.h"
-#include "software/sensor_fusion/ball_detection.h"
-#include "software/sensor_fusion/robot_detection.h"
 #include "software/time/timestamp.h"
-#include "software/world/world.h"
 
```

Comment:
Thoughts on putting the `vision_detection` file in the `proto/message_translation`? Not sure if I'd prefer it there or where it is now.

On one hand makes sense for SensorFusion to define and "own" the datatype it accepts, but also makes sense for the translation code to define the output datatype because it "knows" more what data is available.

Reply:
In general, I think translation code should translate into internal data types that are defined elsewhere in the codebase, rather than define an output type that the rest of the codebase has to accommodate
Vision detection structs are used by filters, so I'll move them there